test-data/pointer.yaml

```
@@ -0,0 +1,356 @@
+    w1 = - w1
+    w1 = - w1
+    r2 = *(u32 *)(r1 + 0)
+
+post:
+  - "r1.type=ctx"
+
+messages:
+  - "0: Invalid type (r1.type == number)"
+  - "1: Invalid type (r1.type == number)"
+  - "2: Lower bound must be at least 0 (valid_access(r1.offset, width=4) for read)"
+  - "2: Upper bound must be at most 64 (valid_access(r1.offset, width=4) for read)"
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)